Gopkg.toml

```diff
@@ -0,0 +1,31 @@
+[[constraint]]
```

Comment:
You don't need to pin (or say "master") versions in this file unless you absolutely must for some reason. Versions are pinned by the lock file.

Reply:
addressed in the last commit.